src/App.tsx

```diff
@@ -21,6 +39,8 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            onChange={changePages}
+            value={itemsPerPage}
           >
             <option value="3">3</option>
             <option value="5">5</option>
```

Comment:
for options better to create array with values and map it.
```const options = [3, 5, 10 20]
```

```suggestion
            {options.map(value => <option key={value} value={value}>{value}</option>)}
```